orca-sql/src/test/java/com/netflix/spinnaker/orca/sql/SqlTestUtil.java

```diff
@@ -33,17 +33,17 @@
 import java.sql.SQLException;
 import java.util.Arrays;
 
-import static org.jooq.SQLDialect.H2;
+import static org.jooq.SQLDialect.MYSQL;
 import static org.jooq.conf.RenderNameStyle.AS_IS;
 
 public class SqlTestUtil {
 
   public static TestDatabase initDatabase() {
-    return initDatabase("jdbc:h2:mem:orca_test");
+    return initDatabase("jdbc:h2:mem:orca_test;MODE=MYSQL");
```

Comment:
?

Reply:
so this was needed otherwise the test wouldn't actually throw bad sql grammar exception (without my fix)
this puts H2 into MySQL compatibility mode and then it throws as expected

Reply:
👍